packages/components/Molecules/Form/__tests__/RplSelect.test.js

```diff
@@ -60,68 +63,52 @@ describe('RplSelect', () => {
     expect(wrapper.find('.rpl-select__native').isVisible()).toBeTruthy()
   })
 
-  it('does not have native select on desktop', () => {
+  it('opens on click', async () => {
```

Comment:
Changes below are just updating test with trigger events to use async await. Jest unit test fails on trigger events without having async await.